src/content/reference/manifest.mdx

```diff
@@ -267,6 +282,22 @@ destroy:
   - terraform destroy --auto-approve
 ```
 
+
+#### Destroy remotely
+
+Remote destroy is useful to share the same tools and configuration across the team.
+
+This will provide all the members in the team the same destroy experience since they are sharing the config and deploy tools defined in the image provided.
```

Comment:
```suggestion
Using the `--deploy` flag lets you run the `okteto deploy` commands directly in your Okteto cluster. 
When executing `okteto destroy` with this flag, Okteto will automatically synchronize your local folder, and then it will run the `destroy` section of the manifest directly in your Okteto cluster, using the image defined in the manifest. 

With this, you don't have to worry about installing tools like `helm`, `kubectl`, and others on your local machine. Instead, you define a container image with all your tools, and you, and everyone else on your team, gets the same experience with zero configuration required. ```